config/application.rb

```diff
@@ -24,5 +24,6 @@ class Application < Rails::Application
     config.active_record.raise_in_transactional_callbacks = true
 
     config.application_name = 'Library Hours'
+    config.pinned_library_slug = 'green'
```

Comment:
I saw that this application uses Rails config. Should we add the config there (for a quick switch in prod)?